### PR TITLE
Default adapter

### PR DESCRIPTION
**Pull request by mtejedorwolox**
## Summary

1. Add default adapter config option
2. Prevents multiple count queries con active record adapter

### CODE REVIEW

File: lib/wor/paginate/adapters/helpers/total_count.rb [hunk 1]
```diff
@@ -4,11 +4,10 @@ module Adapters
       module Helpers
         module TotalCount
           def total_count
-            content = @content.reorder(nil)
-            content_size = content.size
-            return content.to_a.size if content_size.is_a? Hash
+            @content_size ||= @content.reorder(nil).size
```
Comment (wleoguty91) [inline lib/wor/paginate/adapters/helpers/total_count.rb]: Is that ok? I mean, if change the size, you can't view the new size :thinking:

Comment (mtejedorwolox) [reply lib/wor/paginate/adapters/helpers/total_count.rb]: with every pagination the gem creates a new instance of the adapter

File: .rubocop.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ AllCops:
     - spec/dummy/db/**/*
     - spec/dummy/config/**/*
     - spec/spec_helper.rb
+  TargetRubyVersion: 2.3.8
```
Comment (icoluccio) [inline .rubocop.yml]: Shouldn't we use something like 2.6.5?

Comment (mtejedorwolox) [reply .rubocop.yml]: That's the oldest version that the CI uses

**Pull request merged**